lib/services/platform-service.ts

```diff
@@ -416,10 +417,13 @@ export class PlatformService extends EventEmitter implements IPlatformService {
 		await this.trackActionForPlatform({ action: "Build", platform, isForDevice });
 
 		let platformData = this.$platformsData.getPlatformData(platform, projectData);
-		platformData.platformProjectService.on(constants.BUILD_OUTPUT_EVENT_NAME, (data: any) => {
+		const handler = (data: any) => {
 			this.emit(constants.BUILD_OUTPUT_EVENT_NAME, data);
-		});
-		await platformData.platformProjectService.buildProject(platformData.projectRoot, projectData, buildConfig);
+			this.$logger.printInfoMessageOnSameLine(data.data.toString());
```

Comment:
We need in order to keep the current behavior when using `tns` as a CLI.
Whenever we spawn a child process with `pipe` instead of `inherit` as its stdio, we will not see the output on the console.
In short you have 2 cases:
1. You spawn from Fusion(`tns` is `require`-d) - then you spawn with `pipe` and you need this line in order to see the output on the console and you need the event handler in order to capture output and forward it to fusion
2. You spawn from CLI (`tns` executed from command line) - you then spawn with `inherit`, this event handler will not execute because all of the output is printed on `process.stdout`. 

Bottom line - you need this in the unlikely case that you spawn with `pipe` from the terminal so that you see the build output on the console.